models/productDao.js

```diff
@@ -0,0 +1,44 @@
+const appDataSource = require("./appDataSource");
+
+const getProducts = async () => {
+  return await appDataSource.query(
+    `SELECT
+      name,
+      price,
+      thumbnail_image
+      FROM products`
+  );
+};
+
+const getProductInfo = async (productId) => {
+  return await appDataSource.query(
+    `SELECT
+      name,
+      date,
+      price,
+      description,
+      thumbnail_image,
+      product_images.image_url
+      FROM products
+      join product_images on products.id = product_images.product_id
+      where product_id = ?`,
+    [productId]
+  );
+};
```

Comment:
```suggestion
const getProductInfo = async (productId) => {
  return appDataSource.query(
    `SELECT
        p.name,
        p.date,
        p.price,
        p.description,
        p.thumbnail_image,
        pi.image_url
      FROM products p
      JOIN product_images pi ON p.id = pi.product_id
      WHERE pi.product_id = ?`,
    [productId]
  );
};
```
SQL에서 alias(as는 생략가능합니다.)를 활용하여 가독성을 올림과 동시에 해당 컬럼이 어느 테이블에 있는 column인지 구체적으로 명시해줌으로써 발생할 수 있는 에러를 최소화시켜주세요.

Reply:
네 확인했습니다.